docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
 
-### How many validators will be validating at Mainnet launch?
-
-Most likely somewhere between 50 - 100, increasing to 1000 or so later on. [Source](https://youtu.be/IRc5Jma_eH8?t=1630) The number of validators can be changed by governance, and so it is difficult to state precisely how many will be in the active validator set at any particular point in the future.
+## Polkadot Launch
+
+### What is the launch process of Polkadot Beta?
+
+The Polkadot network will launch first in as a Proof-of-Authority (PoA) chain.
+During this time, governance will be controlled by the single Sudo (super-user)
+account, and functionality will be restricted to (among other things) disallow
+the transfer of funds. 
+
+During this time, validators can start joining the network by signalling
+their intention to participate in consensus. When a sufficient number of validator
+are sourced from the community, the Sudo account will initiate the transition
+of the chain security from PoA to Proof-of-Stake (PoS). After this happens,
+the first validator elections will begin and the chain  will be going forward 
+secured by the economic stake that is bonded by validators and nominators.
+
+After the chain has been secured by the decentralized community of validators,
+the next step is to transition the governance of the chain into the hands of the
+token holders. The Sudo account will now initiate an upgrade that erases the
+Sudo logic from the chain entirely and replaces it with the stakeholder
+governance modules. From henceforward, the network is in the hands of the token
+holders and no longer under control of an authority.
+
+The final step to transition to full-functioning Polkadot is the enabling of
+transfer functionality. The community will need to propose a new upgrade and vote
+upon it in order to accomplish this. If the vote to enable transfers passes, then
+it will be enacted shortly thereafter. When this takes place, Polkadot will
+be fully live and move out of it's Beta release label.
+
+### How many validators will be validating at Polkadot Beta launch?
+
+Initially only a few authorized validators will be securing the network. However,
+as [detailed in the answer above](#what-is-the-launch-process-of-polkadot-beta),
+the network will right away be available to vaidators that want to register
+their intention to validate. The transition to Proof-of-Stake will largely
+depend on when a sufficient number of validators have registered and are ready
+to take over the security of the network. This number can be as low as 50 but
+probably closer to 100.
```

Comment:
We may want to reference Kusama and also note that the idea if to increase the number of validators as time goes on.